independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeResourceDeployment.java

```diff
@@ -487,6 +489,31 @@ public RuntimeResource buildResourceMethod(ResourceClass clazz,
                 clazz.resourceExceptionMapper());
     }
 
+    private static void initQuarkusConverter(int i, ParameterConverter quarkusConverter,
```

Comment:
what we're doing here is to initialize the parameter converter, nothing specific for Quarkus, right? 
What I mean is that the method name should be `initParameterConverter` and the `quarkusConverter` field to `parameterConverter`.

Reply:
How about now?

Reply:
+1